posthog/mixins.py

```diff
@@ -11,16 +131,28 @@ class AnalyticsDestroyModelMixin:
     but deletion (i.e. `destroy`) is performed directly in the viewset, which is why this mixin is a thing.
     """
 
-    def perform_destroy(self, instance):
-        instance.delete()
-
-    def destroy(self, request, *args, **kwgars):
+    def destroy(self, request, *args, **kwargs):
 
         instance = self.get_object()  # type: ignore
 
         metadata = instance.get_analytics_metadata() if hasattr(instance, "get_analytics_metadata",) else {}
 
-        self.perform_destroy(instance)
+        instance.delete()
 
         report_user_action(request.user, f"{instance._meta.verbose_name} deleted", metadata)
+
+        if _should_log_history(instance) and metadata:
+            """
+            This is mixed in to API view sets so has team_id available
+            TODO handle models with organization id not team id
```

Comment:
What does that TODO mean? It's not understandable.

Reply:
It's only a note to self about this https://github.com/PostHog/posthog/issues/8545#issuecomment-1035429062

I'll remove it